doc/admin-guide/files/records.config.en.rst

```diff
@@ -3245,6 +3245,10 @@ SSL Termination
 
    Enables (``1``) or disables (``0``) TLS v1.2.  If not specified, enabled by default.  [Requires OpenSSL v1.0.1 and higher]
 
+.. ts:cv:: CONFIG proxy.config.ssl.TLSv1_3 INT 1
+
+   Enables (``1``) or disables (``0``) TLS v1.3.  If not specified, enabled by default.  [Requires OpenSSL v1.1.1 and higher]
```

Comment:
WRT to @maskit this should be updated to note that QUIC depends on TLS 1.3 or better.